myapp/spec/system/tasks_spec.rb

```diff
@@ -48,100 +51,158 @@
 
         # redirected back to root page
         expect(current_path).to eq root_path
-        expect(page).to have_content TasksController::MSG_CREATE_SUCCESS
+        expect(page).to have_content I18n.t 'msg_create_success'
```

Comment:
I'm sorry for lack the explain.

In the case rspec, You should better input string of result directly. like following source-code

```suggestion
        expect(page).to have_content '作成に成功しました'
```

Reply:
Ah ok, it should be plain sting comparison, I understand it.
I just fixed it with some extra tests. https://github.com/Fablic/training/pull/1706/commits/d3b7853281042e717e446bb1df134d101377681d